src/Microsoft.Identity.Client/Cache/StorageJsonValues.cs

```diff
@@ -36,6 +36,7 @@ internal static class StorageJsonValues
         public const string CredentialTypeRefreshToken = "RefreshToken";
         public const string CredentialTypeAccessToken = "AccessToken";
         public const string CredentialTypeIdToken = "IdToken";
+        public const string AccountRootKey = "Account";
         public const string CredentialTypeOther = "Other";
```

Comment:
Does these align with the spec and Python?

Reply:
It would be useful to update the spec, indeed. To be sure everybody is aligned.